crates/rpc-types-engine/src/sidecar.rs

```diff
@@ -51,6 +51,10 @@ impl ExecutionPayloadSidecar {
 
     /// Returns the EIP-7685 requests
     pub const fn requests(&self) -> Option<&Requests> {
-        self.prague.as_ref()
+        if let Some(PragueRequests::Requests(ref requests)) = self.prague {
```

Comment:
please update the doc comment to reflect that this might not be requests anymore

Reply:
this also isn't super helpful for the motivation of the issue; we should probably add a `requests_hash` method as well on the sidecar.